app/pages/sections/order-items/catalogue-solutions/order-item/contextCreator.js

```diff
@@ -15,13 +15,17 @@ export const getContext = ({
   selectedPrice,
   selectedRecipients,
   errorMap,
+  catalogueItemExists,
 }) => {
   const errorMessages = errorMap && (errorMap.quantity || errorMap.deliveryDate)
     ? ((errorMap.quantity || {}).errorMessages || [''])
       .concat((errorMap.deliveryDate || {}).errorMessages) : undefined;
   const newbackLinkRef = selectedPrice.provisioningType === 'Patient'
```

Comment:
I know this `const` was added as part of a previous change but I think we should also rename it to `newItemBackLink` to keep it consistent with `existingItemBackLink`.